Extensions/xinbox.js

```diff
@@ -92,6 +92,22 @@ XKit.extensions.xinbox = new Object({
 			default: true,
 			value: true
 		},
+		"color_code_asks": {
+			text: "color code asks and submissions based on what blog they're for",
+			default: true,
+			value: true,
+		},
+		"color_code_pallet": {
+			type: "text",
+			text: "customize the color coding pallet. <br>"+
+				  'credit for default to <a style="text-decoration: underline;" href="http://colorbrewer2.org/">colorbrewer, by Cynthia Brewer</a>',
```

Comment:
this doesn't style so well but i'm unsure about how to best add the attribution. thoughts?